repose-aggregator/core/repose-core/src/main/java/org/openrepose/powerfilter/PowerFilterChain.java

```diff
@@ -78,16 +79,19 @@
     private RequestTracer tracer = null;
     private boolean filterChainAvailable;
     private TimerByCategory filterTimer;
+    private Optional<String> bypassUrl;
 
     public PowerFilterChain(List<FilterContext> filterChainCopy,
                             FilterChain containerFilterChain,
                             PowerFilterRouter router,
-                            MetricsService metricsService)
+                            MetricsService metricsService,
+                            Optional<String> bypassUrl)
```

Comment:
Oddly, IntelliJ warns on using the `Optional` type for a field or property. I wonder why...